openmetadata-ui/src/main/resources/ui/src/constants/globalSettings.constants.ts

```diff
@@ -14,15 +14,15 @@
 export const GLOBAL_SETTINGS_MENU = [
   {
     category: 'Access',
-    items: ['Teams', 'Users', 'Roles', 'Policies'],
+    items: ['Teams', 'Users', 'Roles'],
   },
   {
     category: 'Services',
     items: ['Databases', 'Messaging', 'Dashboards', 'Pipelines', 'ML Models'],
   },
   {
     category: 'Custom Attributes',
-    items: ['Tables'],
+    items: ['Table', 'Topic', 'Dashboard', 'Pipeline', 'ML Model'],
```

Comment:
```suggestion
    items: ['Tables', 'Topics', 'Dashboards', 'Pipelines', 'MlModels'],
```

<img width="286" alt="image" src="https://user-images.githubusercontent.com/59080942/181236425-b903d2ae-bc97-4711-a333-44079ca4aac7.png">

Reply:
This change is intentional as we are not showing multiple tables/topics etc there, this is section is to only indicate schema of table, topic etc that's why, let me know if I am missing anything!

Reply:
@ShaileshParmar11, Here it represents all the entities in OM, so it should be like that.

Reply:
If you see the custom properties page screenshot that I provided has the same thing.